File: Kotlin/PraticandoAritmetica/EasyFibonacci.kt
fun main(args: Array<String>) {
    
    val n = readLine()!!.toInt()
    var t1: Int = 0
    var t2: Int = 1
    var sum : Int
    
    for (i in 1..(n-1)) {
        print("$t1 ")
        
        sum = t1 + t2 
        t1 = t2 
        t2 = sum
    }
    print("$t1")
    println()
    
}

File: JS/LocalizaLabsReactDev/DesafiosAritmeticosEmJS/EasyFibonacci.js
let t1 = 0
let t2 = 1
let sum = 0

let n = gets()
let str = ""

for (let i = 0; i < n-1; ++i) {
  str += t1 + " ";
        
  sum = t1 + t2; 
  t1 = t2; 
  t2 = sum;
}
    
str += t1
console.log(str)

